devito/arch/compiler.py

```diff
@@ -397,7 +397,7 @@ def __init__(self, *args, **kwargs):
             # Add flags for OpenMP offloading
             if language in ['C', 'openmp']:
                 # TODO: Need a generic -march setup
-                # self.cflags += ['-Xopenmp-target', '-march=sm_37']
+                self.cflags += ['-Xopenmp-target', '-march=sm_'+str(detect_cc())]
```

Comment:
join str

Reply:
as you know, this is not the way we do string concatenation in devito